### DISCUSSION TITLE
[feat] Supply developers with tools to help develop Tauri apps securely on their machine

### DISCUSSION THREAD

**Discussion by chip-crabnebula** [tauri-apps/tauri#7825] [Ideas]
### Describe the problem

We have a decent page listing [Development Security](https://tauri.app/v1/references/security) threats and we should help provide tooling to enable developers to develop more securely not only for the downstream users, but for their own security. Things like dedicated VM images, isolation from network/disk, and other development threats that target developer machines.

### Describe the solution you'd like

Tools that help protect the developer and their machine from development threats.

**Comment by JonasKruckenberg** [comment]
Ahh look who's there ![image](https://github.com/tauri-apps/tauri/assets/25329995/84cc4783-5f59-42bd-8dc8-78c184344419)

Edit: while I wholeheartedly agree - this is way to vague it should be a discussion instead
